### PR TITLE
refactor export mercator users script

### PR DESCRIPTION
**Pull request by joka**

### CODE REVIEW

**Comment by joka** [comment]
@MagdaN @nidico  @pallix  I did some refactoring in scripts/export_users.py.
Now  it follows the style guide better and gives a good example how to use the ipool sheet for catalog searches and to use argparse instead of optparse (outdated).
Also the performance is much better, even if there is still room for improvement.

**Comment by nidico** [comment]
I just quickly scanned the changes and they look good, but @pallix should look in more depth into it.

File: src/adhocracy_mercator/adhocracy_mercator/scripts/export_proposals.py [hunk 1]
```diff
@@ -1,20 +1,18 @@
-"""Export scripts for adhocracy_mercator.
+"""Export proposal.
```
Comment (pallix) [inline src/adhocracy_mercator/adhocracy_mercator/scripts/export_proposals.py]: @slomo need to update the cron script on the production server once this branch is merged

**Comment by pallix** [comment]
Looks good

**Comment by pallix** [comment]
Can you merge master?

**Comment by joka** [comment]
master is merged

**Pull request merged**